docs/source/welcome_to_sycamore/get_started.md

```diff
@@ -121,6 +119,50 @@ This will crawl and download the data from the specified website. If you import
 If you want to prepare you data with custom code, you can use a Jupyter notebook to iterate and test it. NEED LINK
 
 
+## Add a dataset from a S3 bucket
+
+You can try using an arbitrary PRF and HTML data from an S3 bucket with the Sort Benchmark
+importing script. This script is not optimized for new datasets, so the answer quality may
+vary. However we have found positive results with some experiments.
+
+WARNING: Processing data using the Sort Benchmark importing script will send your data to OpenAI,
+and optionally Amazon Textract for calls to their AI services. Consider whether this is acceptable
+if you are using a non-public website for testing.
+
+1. Run the Sycamore S3 Crawler container with one or two additional parameters:
```

Comment:
Either improve the instructions to say you can leave off the prefix:
```
# load every file in a bucket:
docker compose run sycamore_crawler_s3 _bucket_
# for example
docker compose run sycamore_crawler_s3 aryn-public
```
or remove the one or two from the line above as we only give two argument examples.

Reply:
Ack